libkernelflinger/tpm2_security.c

```diff
@@ -453,6 +453,16 @@ static EFI_STATUS check_provision_status(void)
 	return EFI_SUCCESS;
 }
 
+EFI_STATUS tpm2_delete_index(UINT32 index)
+{
+	EFI_STATUS ret = Tpm2NvUndefineSpace(TPM_RH_OWNER, index, NULL);
+
+	if (EFI_ERROR(ret))
+		error(L"Delete TPM NV index failed, index: %x, ret: %d", index, ret);
```

Comment:
Use `efi_perror()`

Reply:
I see that you have not followed good practices for a while.  I created a fix and a merge request, please review.
clean-up: use efi_perror() instead of error() when appropriate #34